physionet-django/project/views.py

```diff
@@ -56,6 +56,13 @@
 from user.forms import AssociatedEmailChoiceForm
 from user.models import CloudInformation, CredentialApplication, LegacyCredential, User, Training
 
+from .serializers import PublishedProjectSerializer, PublishedProjectDetailSerializer
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
```

Comment:
`from rest_framework.permissions import IsAuthenticated` looks like it isn't used?